pkg/config/config.go

```diff
@@ -59,6 +61,7 @@ func NewConfig() (*Config, error) {
 		NodePool:        viper.GetString("nodepool"),
 		ForceAssignment: viper.GetBool("forceassignment"),
 		AdditionalNodePools: AdditionalNodePools,
+		Ticker:           viper.GetInt("ticker"),
```

Comment:
Isn't it best to use `GetDuration(key string) : time.Duration`? 
https://godoc.org/github.com/spf13/viper#GetDuration